Shopping List/Model/ShoppingItemController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ShoppingItemController.swift
+//  Shopping List
+//
+//  Created by Cora Jacobson on 6/13/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ShoppingItemController {
+    
+    init() {
+        if UserDefaults.standard.bool(forKey: "shoppingItemsInitialized") {
+            loadFromPersistentStore()
+        } else {
+        for item in itemNames {
+            shoppingItems.append(ShoppingItem(name: item, addedToList: false, imageName: item))
+        }
+        saveToPersistentStore()
+        UserDefaults.standard.set(true, forKey: "shoppingItemsInitialized")
+        }
+    }
+    
+    let itemNames = ["Apple", "Grapes", "Milk", "Muffin", "Popcorn", "Soda", "Strawberries"]
+    
+    var shoppingItems: [ShoppingItem] = []
+    
+    func itemWasUpdated() {
+        saveToPersistentStore()
+    }
+    
+    var addedItems: [ShoppingItem] {
+        let addedItemsArray = shoppingItems.filter { $0.addedToList == true }
+        return addedItemsArray.sorted(by: { $0.name < $1.name })
+    }
+    
+    var unaddedItems: [ShoppingItem] {
+        let unaddedItemsArray = shoppingItems.filter { $0.addedToList == false }
+        return unaddedItemsArray.sorted(by: { $0.name < $1.name })
+    }
```

Comment:
Very nice! This provides a nice clean interface to access the added and un-added items in the list.